tiatoolbox/tools/tissuemask.py

```diff
@@ -0,0 +1,289 @@
+# ***** BEGIN GPL LICENSE BLOCK *****
+#
+# This program is free software; you can redistribute it and/or
+# modify it under the terms of the GNU General Public License
+# as published by the Free Software Foundation; either version 2
+# of the License, or (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program; if not, write to the Free Software Foundation,
+# Inc., 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301, USA.
+#
+# This file contains code inspired by StainTools
+# [https://github.com/Peter554/StainTools] written by Peter Byfield.
+#
+# The Original Code is Copyright (C) 2020, TIALab, University of Warwick
+# All rights reserved.
+# ***** END GPL LICENSE BLOCK *****
+
+"""Methods of masking tissue and background."""
+
+from abc import ABC, abstractmethod
+
+import cv2
+import numpy as np
+from skimage.filters import threshold_otsu
+
+from tiatoolbox.utils.misc import objective_power2mpp
+
+
+class TissueMasker(ABC):
+    """Base class for tissue maskers.
+
+    Takes an image as in put and outputs a mask.
+    """
```

Comment:
Add a space at the end.

Reply:
Fixed in this commit https://github.com/TIA-Lab/tiatoolbox/pull/61/commits/5b0e0ecc1fcb9ce989d907483bef9a2b0b2c7aa5